po/fr.po

```diff
@@ -78,8 +78,8 @@ msgid ""
 "Temporarily show the application numbers over the icons, corresponding to "
 "the shortcut."
 msgstr ""
-"Afficher temporairement les numéros d'application sur les icônes, correspondant"
-"au raccourci clavier."
+"Afficher temporairement les numéros d'application sur les icônes, "
+"correspondantau raccourci clavier."
```

Comment:
@micheleg there's one missing space, I don't think it's worth a PR so maybe you could fix it directly ?